sentry_sdk/integrations/django/middleware.py

```diff
@@ -34,10 +34,11 @@
     import_string_name = "import_string"
 
 
-if DJANGO_VERSION < (3, 1):
-    _asgi_middleware_mixin_factory = lambda _: object
-else:
+django_supports_async_middleware = DJANGO_VERSION >= (3, 1)
```

Comment:
Please use all caps variable name for constants like this one, i.e. `DJANGO_SUPPORTS_ASYNC_MIDDLEWARE`

Reply:
I've applied `black` and adjusted the variable case.

I'm puzzling over the CI failure here:
https://github.com/getsentry/sentry-python/actions/runs/6660942801/job/18340882553

It looks like the version of pytest is too old for upcoming versions of pytest-asyncio. However, I don't see how this should happen only on my branch and not also on the main sentry codebase. I've merged recent commits from master into my branch, but none seems relevant to this test so I don't imagine the next CI run should be any better. But please push the button on it just for the exercise.

I'll keep looking a bit at this today. Is there a guide to running these tests locally?